### ISSUE TITLE
Param Mismatch

### ISSUE THREAD

**Issue by rdslater-tc** [obryanlouis/qa#3]
Having a bit of trouble getting things running.

Setup via anaconda (except cove/and this repository)
 Using Python 3.6
Setup works smooth

First initial train getting errors: (inside a condo environ with python 3)
python train_local.py python train_local.py --model_type=fusion_net --rnn_size=64 --batch_size=8 --input_dropout=0.4 --rnn_dropout=0.3 --dropout=0.4 --num_gpus=1

tensorflow/contrib/cudnn_rnn/kernels/cudnn_rnn_ops.cc:433] Check failed: params_input.size() == num_params Number of params mismatch. Expected 16, got 8

The full output looks like this:

`Vocab size: 2196016`
`1 train files batches, 1 dev`
`Creating towers`
`Creating tower in model`
`Tower created`
`Time to create towers: 2.3235864639282227`
`Time to compute gradients: 1.5606577396392822`
`Time to create towers and calculate gradients: 3.936843156814575`
`Applying gradients`
`Time to apply gradients: 2.0887742042541504`
`Restoring or creating new model...`
`Creating model with new parameters`
`Model initialization time 5.168830394744873`
`Setting up tensorflow data iterator handles`
`Total setup time before starting training: 29.380147457122803`
`Current iteration: 1, Iters/epoch: 10949, Current learning rate: 0.001000`
`2018-01-12 21:50:18.272070: F tensorflow/contrib/cudnn_rnn/kernels/cudnn_rnn_ops.cc:433] Check failed: params_input.size() == num_params Number of params mismatch. Expected 16, got 8`
`Aborted (core dumped)`

I tracked it to about line 140 in trainer.py but now amount of flags/parameter shifting gave me anymore clues--changes batch size, run size did not affect the error.

**Comment by obryanlouis** [comment]
Can you try in the following order:
1) Batch size 1 (to rule out memory-related errors)
2) Restart the computer (I have seen this sometimes with a similar message related to CUDA)

Can you also verify that you are on tf 1.3 or 1.4?

**Comment by rdslater-tc** [comment]
appreciate the follow up.

Both of those suggestions failed (produced the same error).  However, since we had it working yesterday and I remember I bumbled the TensorFlow install (Working on spot instances currently so they disappear when not in use), I looked at the TF ver (1.3).  On a hunch I upgraded to 1.4.1 and the issue was resolved.

Excellent work on the library btw.  Very much appreciated.
